moha/rauk/rauk.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np 
+
+from scipy import constants 
+
+from scipy.special import gamma
+
+from moha.utils import get_atom_type
+
+from scipy.sparse import csr_matrix, diags
+import os
+import json
+
+def build_one_body_matrix(connectivity, atom_dictionary, atoms_num, n_sites, bond_dictionary):
+    # Populate diagonal and non-diagonal matrix elements
+    param_diag_mtrx = np.zeros((n_sites, n_sites))
+    for atom, site in atoms_num:
+        key = atom
+        param_diag_mtrx[site - 1, site - 1] = atom_dictionary[key]
+    print(param_diag_mtrx)
+
+    param_nodiag_mtrx = np.zeros((n_sites, n_sites))
+    for atom1, atom2, _ in connectivity:
+        atom1_name, site1 = get_atom_type(atom1)
+        atom2_name, site2 = get_atom_type(atom2)
+        key1 = atom1_name #if atom1_coord else atom1_name
+        key2 = atom2_name #if atom2_coord else atom2_name
+
+        #print((site1 - 1, site2 - 1), key1 + key2, self.bond_dictionary[key1 + key2])
+        param_nodiag_mtrx[site1 - 1, site2 - 1] = bond_dictionary[key1 + key2]
+        
+    param_nodiag_mtrx = np.minimum(param_nodiag_mtrx, param_nodiag_mtrx.T)
+    param_nodiag_mtrx = param_nodiag_mtrx
+    param_diag_mtrx = csr_matrix(param_diag_mtrx)
+    param_nodiag_mtrx = csr_matrix(param_nodiag_mtrx)
+    
+    one_body = param_diag_mtrx + param_nodiag_mtrx
+    
+    return one_body
+
+def assign_rauk_parameters(connectivity, atom_types, atoms_num, n_sites):
+    r"""Assigns the alpha and beta value from Rauk's table in matrix form"""
+
+    dir_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
it's not the best practice to use absolute pathway for the file, because it may not work when package is installed and in use. Instead, we can use `Path` from a [`pathlib` standard python library](https://docs.python.org/3/library/pathlib.html). [Here](https://github.com/theochem/ModelHamiltonian/blob/28a97bde0a88407547de2b37fe793cffa5fb5ce3/moha/gpt/gpt.py#L155) is an example on how we used it in this library before